### PR TITLE
Add Methods:

### PR DESCRIPTION
**Pull request by amit-jain**
- Add overloaded method to activate/deactivate based on agentId filter
- checkContentDistributionAgentExists - to check the given agent exists
- waitQueueEmptyOfPath - Checks the queue at the given queuepath is empty or doesn't contain the path

### CODE REVIEW

**Comment by amit-jain** [comment]
@dulvac Appreciate your review

File: src/main/java/com/adobe/cq/testing/client/ReplicationClient.java [hunk 1]
```diff
@@ -225,6 +267,112 @@ public JsonNode getAgentReplicationQueue(String agentPath) throws ClientExceptio
...
+        if (agents.path(agent).isMissingNode()) {
+            log.warn("Default distribution agent {} is missing from the distribution list", agent);
+            return false;
+        }
+        return true;
+    }
+    
+    /**
+     * Checks if the item is empty or if not whether the replicated path is in the list of packages.
+     * Sample curl response
```
Comment (dulvac) [inline src/main/java/com/adobe/cq/testing/client/ReplicationClient.java]:
```suggestion
     * Sample HTTP response
```

**Review by dulvac — APPROVED** [review]

**Comment by dulvac** [comment]
@volteanu can you help with a release here, please?

**Comment by volteanu** [comment]
released in aem-cloud-testing-clients-0.8.2

**Comment by amit-jain** [comment]
Thanks @volteanu & @dulvac

**Pull request merged**